gitops/generate_build.go

```diff
@@ -97,7 +118,8 @@ func DetermineBuildExecution(component appstudiov1alpha1.Component, params []tek
 			{
 				Name: "registry-auth",
 				Secret: &corev1.SecretVolumeSource{
-					SecretName: "redhat-appstudio-registry-pull-secret",
+					SecretName: component.Name,
+					// TODO: Cleanup "redhat-appstudio-registry-pull-secret" from namespaces,
```

Comment:
Do we need to care about this before GA?

Reply:
We'll be in a different cluster probably by then